addons/mail/static/src/models/chatter.js

```diff
@@ -41,7 +41,12 @@ registerModel({
          * @param {MouseEvent} ev
          */
         onClickButtonAttachments(ev) {
-            this.update({ attachmentBoxView: this.attachmentBoxView ? clear() : insertAndReplace() });
+            if (this.thread && this.thread.isFileBrowserOpen) {
+                this.update({ attachmentBoxView: insertAndReplace() });
+                this.attachmentBoxView.onClickAddAttachment();
```

Comment:
We should never call click handlers manually in code, the current flow is not a click on the attachment box view button.

If upload can now be done without the view being there, the file uploader should be moved up in chatter/chatter top bar.